main/src/main/java/org/mobilitydata/gtfsvalidator/validator/StopNameValidator.java

```diff
@@ -51,6 +63,30 @@ private boolean isValidStopDesc(String stopDesc, String stopName) {
     return !stopDesc.equalsIgnoreCase(stopName);
   }
 
+  /**
+   * {@code stops.stop_name} is required for a {@code GtfsStop} with {@code stops.location_type} of {@code 0}, {@code 1}, or {@code 2}
+   *
+   * <p>"Please provide a stop name for Stops, Stations, and Entrance/Exits."
+   * (http://gtfs.org/reference/static#stopstxt)
+   *
+   * <p>Severity: {@code SeverityLevel.Error}
+   */
+  static class MissingStopNameNotice extends ValidationNotice {
+
+    private final long csvRowNumber;
+    private GtfsLocationType locationType;
+    private final String stopId;
+    private final String routeDesc;
```

Comment:
I'm not sure it's worth including stopDesc here.  It's not really relevant to the name.